ssld/ssld.c

```diff
@@ -19,13 +19,70 @@
  *  USA
  */
 
+/**
+ * IPC format
```

Comment:
This probably belongs in the [wiki](https://github.com/Elemental-IRCd/elemental-ircd/wiki) too.